docs/use_case/post_demo_app_intro.md

```diff
@@ -0,0 +1,53 @@
+In this tutorial we are going to present 3 versions of ONEm's #post application. The apps are written using the Django framework ONEm's Python SDK.
+
+- **postv1** - a stripped down version of the service; we will be able to create, store and view posts
+- **postv2** - on top of v1, we will now add a Search wizard so we can search by keyword/code in all saved posts (ours or others)
+- **postv3** - on top of v2, we will enable interactions with ONEm's restdAPI
+
+For prerequisites and running any of the 3 apps locally please refer to: [prerequisites](/getting_started/python_prereq) and [run locally](/getting_started/python_run_local/) tutorials.
+
+
+Notes: 
+
+1) The project containing these 3 apps can be cloned from here: [**post-demo-app**](https://github.com/ONEmCommunications/post-demo-app)
+or
```

Comment:
Not sure what exactly you mean by this `or`, but I understand that the line below is putting in practice the theory from the line above. So I wouldn't link them with an "or" which, in my view, should separate 2 different alternatives.

Reply:
it stands for either do it like this or like that. If I add `or from the command line:` does it makes more sense ?

Reply:
The only way to **clone** a repo you describe is from command line `git clone <repo>`.
If you want to mention about the alternative to download the repo from the web interface, I guess you have to explicitly explain how to do that (including unarchiving the downloaded file - because downloading is not equivalent with cloning).